org.eclipse.zest.core/src/org/eclipse/zest/core/widgets/internal/XYScaledGraphics.java

```diff
@@ -65,6 +65,13 @@ protected FontKey(Font font, int height) {
 
 		@Override
 		public boolean equals(Object obj) {
+			if (this == obj) {
+				return true;
+			}
+
+			if (obj == null || getClass() != obj.getClass()) {
```

Comment:
Same as before, can FontKey be made into a record?

Reply:
The font key members are updated so I would say for now not.

Reply:
I think it's updated to avoid creating a new instance every time. But it's fine to keep it like this.